concrete/blocks/desktop_app_status/controller.php

```diff
@@ -26,25 +27,22 @@ public function getBlockTypeName()
 
     public function view()
     {
+        $config = $this->app->make('config');
+        $this->set('current_version', $config->get('concrete.version'));
         $this->set('latest_version', Update::getLatestAvailableVersionNumber());
-        $local = [];
-        $remote = [];
-        $p = new Permissions();
-        if ($p->canInstallPackages()) {
-            $local = Package::getLocalUpgradeablePackages();
-            $remote = Package::getRemotelyUpgradeablePackages();
-        }
-
-        // now we strip out any dupes for the total
         $updates = 0;
-        $localHandles = [];
-        foreach ($local as $_pkg) {
-            ++$updates;
-            $localHandles[] = $_pkg->getPackageHandle();
-        }
-        foreach ($remote as $_pkg) {
-            if (!in_array($_pkg->getPackageHandle(), $localHandles)) {
+        $p = new Checker();
+        if ($p->canInstallPackages()) {
```

Comment:
Haven't checked this class, but the scrutinizer is saying something about the canInstallPackages function not existing

Reply:
Yep, the methods of `Concrete\Core\Permission\Checker` are *magic* (and this call hasn't changed in this PR).